doc/BUILD

```
@@ -183,13 +183,23 @@ py_test_run_all_subdirectory(
     size = "large",
     include = ["source/ray-air/doc_code/*.py"],
     exclude = [
+        # Too large. Use a custom test below.
+        "source/ray-air/doc_code/computer_vision.py",
         "source/ray-air/doc_code/hf_trainer.py",  # Too large
         "source/ray-air/doc_code/predictors.py",
     ],
     extra_srcs = [],
     tags = ["exclusive", "team:ml"],
 )
 
+py_test(
+    name = "computer_vision_test",
+    main = "source/ray-air/doc_code/computer_vision.py",
+    srcs = ["source/ray-air/doc_code/computer_vision.py"],
+    tags = ["exclusive", "team:ml"],
+    size = "enormous",
```

Should this be `large`? Since it finishes in 566 < 900 secs.

```suggestion
    size = "large",
```

---

apparently it runs faster once I separated it out.
it was timing out consistently at 900s.

---

I thought we were supposed to avoid using the 'enormous' tag?

---

😕 
so the options are splitting the actual script into 2.
or specify a different more specific limit?
do you have a preference?